### ISSUE TITLE
A white blank small screen <Data Engineer> <Microsoft Bug Bash> <Microsoft Fabric>

### ISSUE THREAD

**Issue by marwashahid** [microsoft/studentambassadors#211]
### Describe the bug

A small white screen keeps on my fabric interface.

### What happened?

I tried to create a new pipeline by clicking on new pipeline but a blank white screen appeared. After waiting for 10-15 minutes, I went back using the chrome's back button, I kept going back to back, clicked on several places but the box remains on the fabric's interface. 

### What did you expect to happen?

I expected the box to disappear.

### How would you reproduce the bug?

1. Create a new pipeline 
2. A blank box appears
3. It doesn't disappear then

### Did this happen via Desktop and/or Smartphone?

Desktop

### Operating System

Windows 10

### What browsers are you seeing the problem on?

Chrome

### Version

latest

### Device

DELL Latitude E7250

### What is your current role?

Student

### What technical topic is this bug related to?

AI/Data Science/Machine Learning

**Comment by marwashahid** [comment]
It got disappeared after the second login.

**Comment by marwashahid** [comment]
I had to select Student Ambassador and accidently selected student.

**Comment by vivsridh4** [comment]
@marwashahid, can you confirm this issue has been resolved?

**Comment by marwashahid** [comment]
@vivsridh4  yes it got resolved after some time, I believe there is a high latency in the fabric.
